src/project.c

```diff
@@ -772,6 +772,53 @@ static gboolean update_config(const PropertyDialogElements *e, gboolean new_proj
 		}
 		g_free(locale_path);
 	}
+
+	/* create filename path if it doesn't exist */
+	if ((err_code = utils_is_file_writable(locale_filename)) != 0) {
```

Comment:
Should follow the style of the other code with respect to braces. There's some style info in the [`HACKING` file](https://github.com/geany/geany/blob/55e0998c3b131e8febe118245778702e65d4086f/HACKING#L271).